### ISSUE TITLE
Reaction log

### ISSUE THREAD

**Issue by shivzy2002** [stem-discord/stembot-issues#25]
Something like this perhaps: ![image](https://user-images.githubusercontent.com/66868622/105956318-79030280-602c-11eb-9568-a46471ac5eda.png)

Channel already made in the logs category

**Comment by nopeless** [comment]
ill make smth

**Comment by nopeless** [comment]
@shivzy2002 send code so i can copy :barry:

**Comment by shivzy2002** [comment]
@nopeless its in python, is that okay?

**Comment by nopeless** [comment]
> @nopeless its in python, is that okay?

@shivzy2002 yes i speks snake

**Comment by shivzy2002** [comment]
its bad code, pls no bully :crycat:
```py
reaction_log_channel_id = 803876328620163072
@client.event
async def on_raw_reaction_add(payload):
    if payload.guild_id != guild_id:
        return
    if payload.user_id == 775652866680946688 or  payload.user_id == 510789298321096704:
        return
    reaction_log_channel = await client.fetch_channel(reaction_log_channel_id)

    guild = client.get_guild(payload.guild_id)
    user = await client.fetch_user(payload.user_id)
    channel = await client.fetch_channel(payload.channel_id)
    message = await channel.fetch_message(payload.message_id)
    
    embed = discord.Embed(
    description=f'**Reaction from** {user.mention} **added in** {channel.mention} [Jump to Message](https://discordapp.com/channels/{payload.guild_id}/{payload.channel_id}/{payload.message_id})\n``Message Content:`` {message.content}')
    embed.set_author(name=f"{user}", icon_url=user.avatar_url)
    embed.set_footer(text=f'Message ID: {payload.message_id} | Reactor ID: {payload.user_id}')
    embed.add_field(name='❯ Reaction', value=f'{payload.emoji}',inline=False)
    await reaction_log_channel.send(embed=embed)

@client.event
async def on_raw_reaction_remove(payload):
    if payload.guild_id != guild_id:
        return
    if payload.user_id == 775652866680946688 or  payload.user_id == 510789298321096704:
        return
    reaction_log_channel = await client.fetch_channel(reaction_log_channel_id)

    guild = client.get_guild(payload.guild_id)
    user = await client.fetch_user(payload.user_id)
    channel = await client.fetch_channel(payload.channel_id)
    message = await channel.fetch_message(payload.message_id)
    
    embed = discord.Embed(
    description=f'**Reaction from** {user.mention} **removed in** {channel.mention} [Jump to Message](https://discordapp.com/channels/{payload.guild_id}/{payload.channel_id}/{payload.message_id})\n``Message Content:`` {message.content}')
    embed.set_author(name=f"{user}", icon_url=user.avatar_url)
    embed.set_footer(text=f'Message ID: {payload.message_id} | Reactor ID: {payload.user_id}')
    embed.add_field(name='❯ Reaction', value=f'{payload.emoji}',inline=False)
    await reaction_log_channel.send(embed=embed)

**Comment by nopeless** [comment]
fixed in version 1.1.1
